developer/guides/troubleshooting/tailwind-styles-being-overridden.mdx

```diff
@@ -0,0 +1,26 @@
+---
+title: Tailwind styles are being overridden
+description: Resolve issues with Tailwind v4 or other @layer-based CSS frameworks being overridden by CSS resets
+---
+
+## Problem
+
+Starting from [version 4](https://tailwindcss.com/blog/tailwindcss-v4-alpha#designed-for-the-modern-web), Tailwind CSS uses `@layer` rules to define its styles.
+
+Makeswift by default includes a CSS reset from [normalize.css](https://github.com/necolas/normalize.css/).
+Because the CSS reset is unlayered, it'll override the Tailwind styles, even if the Tailwind styles are more specific.
+
+From [MDN @layer documentation](https://developer.mozilla.org/en-US/docs/Web/CSS/@layer):
```

Comment:
```suggestion
From [MDN @layer documentation](https://developer.mozilla.org/en-US/docs/Web/CSS/@layer): "Styles that are not defined in a layer always override styles declared in named and anonymous layers."
```

Reply:
Added. Thank you. That's the most important sentence from the link.